webapps/stress-test-center-view/style.css

```diff
@@ -207,20 +207,39 @@ transform: translate(-125%, -50%);
     color: var(--white);
 }
 
-.dku-btn-primary:disabled, .dku-btn-primary:active:hover {
+.dku-btn-primary:disabled, .dku-btn-primary:disabled:active {
     background: var(--blue-lighten-4);
     cursor: not-allowed;
 }
 
 .dku-btn-primary:active {
-    background: #3194fc;
+    background: var(--blue-darken-4);
 }
 
 .dku-btn-container--right-align {
     display: flex;
     justify-content: flex-end;
 }
 
+.dku-btn--txt {
+    color: var(--blue);
+    background: none;
+    border: none;
+}
+
+.dku-btn--txt:hover {
+    background: var(--blue-lighten-5);
+}
+
+.dku-btn--txt:active {
+    background: var(--blue-lighten-4);
+}
+
+.dku-btn--txt:disabled {
```

Comment:
We should probably also remove hover/active style if disabled

Reply:
Yep I agreed! But somehow  when I tried to hover/click on the disabled text button with the CSS code as is, the style was the proper disabled one 🤷‍♀️

Reply:
Done in 66174a03a950082f9db5bd0d7b4d958ef5d406d2